### PR TITLE
Doc deployment

### PR DESCRIPTION
**Pull request by Pipazoul**
- Initial informations on how to run locally / build for unready production

### CODE REVIEW

File: apps/doc/docs/intro.md [hunk 1]
```diff
@@ -2,58 +2,132 @@
...
-- [Node.js](https://nodejs.org/en/download/) version 16.14 or above:
-  - When installing Node.js, you are recommended to check all checkboxes related to dependencies.
+```bash
+docker-compose up -f dev.docker-compose.yml
 
-## Generate a new site
+```
 
-Generate a new Docusaurus site using the **classic template**.
+In another shell, run the backend
```
Comment (lutangar) [inline apps/doc/docs/intro.md:52]: Could/should the backend and/or run in another container instead ?

File: apps/doc/docs/intro.md [hunk 2]
```diff
@@ -2,58 +2,132 @@
...
+- pgadmin at http://localhost:5431
+- the documentation at http://localhost:3001
+
+#### 🔎 Test and lint
+
+Jest, Cypress and husky are used for testing and 🌸 linting.
+
+** 🐺 Husky**
+
+Please install husky on your local machine to run the pre-commit hook
```
Comment (lutangar) [inline apps/doc/docs/intro.md]: This isn't required.

File: apps/doc/docs/intro.md [hunk 3]
```diff
@@ -2,58 +2,132 @@
...
+
+```bash
+npm install -g husky
+```
+
+Use 'nx format:write' before commiting to format the code with prettier
```
Comment (lutangar) [inline apps/doc/docs/intro.md]: This is now done on pre-commit. You could still rephrase to mention this.

File: apps/doc/docs/intro.md [hunk 4]
```diff
@@ -2,58 +2,132 @@
...
+
+**Frontend**
 
 ```bash
-cd my-website
-npm run start
+npx nx run frontend:test
```
Comment (lutangar) [inline apps/doc/docs/intro.md]: Shall we mention the command to run them all instead ?
```
npx nx run-many --target=test
```
We already have many apps and if you mention 2 of them you should probably mention all of them.... instead this command allow to doc to scale to many more apps !

File: apps/doc/docs/intro.md [hunk 5]
```diff
@@ -2,58 +2,132 @@
...
 
-## Generate a new site
+```
 
-Generate a new Docusaurus site using the **classic template**.
+In another shell, run the backend
 
-The classic template will automatically be added to your project after you run the command:
+```bash
+npx nx serve backend
```
Comment (lutangar) [inline apps/doc/docs/intro.md:55]: I wonder if this would work instead :
```
npx nx run-many --target=serve
```

Comment (lutangar) [reply apps/doc/docs/intro.md:55]: And would it run in container ?

**Review by lutangar — CHANGES_REQUESTED** [review]

File: apps/doc/docs/intro.md (hunk 1 above)
Comment (Pipazoul) [reply apps/doc/docs/intro.md:52]: Thats not the case yet but i've created an issue  https://github.com/datatlas-erasme/datatlas/issues/97

File: apps/doc/docs/intro.md (hunk 5 above)
Comment (Pipazoul) [reply apps/doc/docs/intro.md:55]: Added to issue #97 
https://github.com/datatlas-erasme/datatlas/issues/97#issuecomment-1453386331

**Review by lutangar — APPROVED** [review]

**Pull request merged**